server/views/v2Manage/outOfServiceBeds/partials/_bedDetails.njk

```diff
@@ -8,6 +8,9 @@
     {% endfor %}
   </ul>
   {%endset%}
+
+  {% set latestRevision = outOfServiceBed.revisionHistory[0] %}
```

Comment:
the `latestRevision` variable is v helpful but I think we'll need to verify that the most recent revision is the first in the `revisionHistory` list? i.e. that revisions are provided in this order?

Reply:
Good suggestion, [I have checked with Dan](https://mojdt.slack.com/archives/C06HPHL7J8K/p1720439106404229?thread_ts=1720438933.073579&cid=C06HPHL7J8K) and there is currently no sorting performed in the API so I will add some in the UI.